dynatrace_base/setup.py

```diff
@@ -24,10 +24,10 @@
 setup(
     name='stackstate-dynatrace',
     version=ABOUT['__version__'],
-    description='The Dynatrace check',
+    description='The Dynatrace_base check',
```

Comment:
Not a check but `Dynatrace base library`

Reply:
Done